### PR TITLE
[minor] Update `skylinkUrl` docstrings

### PR DESCRIPTION
**Pull request by m-cat**
# PULL REQUEST

## Overview

The docstrings should be more clear than "Skylink striing".

### CODE REVIEW

File: src/download.ts [hunk 1]
```diff
@@ -180,7 +180,7 @@ export async function downloadFileHns(
  * Constructs the full URL for the given skylink.
  *
  * @param this - SkynetClient
- * @param skylinkUrl - Skylink string. See `downloadFile`.
+ * @param skylinkUrl - Base64 skylink, or a valid URL that contains a skylink. See `downloadFile`.
```
Comment (kwypchlo) [inline src/download.ts:183]: does it also accept base 32 skylink? I think it should right?

Comment (m-cat) [reply src/download.ts:183]: Nope. `parseSkylink` only detects base32 if the `fromSubdomain` option is set and the input is a URL. That method probably needs an overhaul to automatically detect the skylink type.

Related: https://github.com/SkynetLabs/skynet-js/issues/238

Comment (m-cat) [reply src/download.ts:183]: More directly related: https://github.com/SkynetLabs/skynet-js/issues/237

Just haven't had time to make this change I guess.

**Review by kwypchlo — APPROVED** [review]

**Review by ro-tex — APPROVED** [review]
This is great, thank you for improving the docs!

**Pull request merged**